tests/utils/parseDate.unit.test.js

```diff
@@ -0,0 +1,24 @@
+test('Test date format parsing', () => {
```

Comment:
This file should probably better be formatted this way (easier error parsing when tests fail)
I also normally but test files as siblings of the file they are testing, as it's more modular that way.

```
const utils = require('../../app/utils/parseDate.js'); // you should be able to use an import here

describe('date format parsing', () => {
  it('do date formats validate', () => {});
  it('do dates parse correctly', () => {});
  it('do timestamps format correctly', () => {});
  it('does disproveList work properly', () => {});
});
```

Also be sure to test *ALL* failing states as well, you never know when something might actually pass that you didn't expect.